lib/cocina/models/file.rb

```diff
@@ -22,6 +22,8 @@ class File < Struct
       attribute :version, Types::Strict::Integer
       # MIME Type of the File.
       attribute? :hasMimeType, Types::Strict::String
+      # BCP 47 language tag: https://www.rfc-editor.org/rfc/rfc4646.txt -- other applications (like media players) expect language codes of this format, see e.g. https://videojs.com/guides/text-tracks/#srclang
+      attribute? :languageTag, Types::Strict::String.optional
```

Comment:
i wasn't sure whether `.optional` was redundant given `attribute?`, though i see `presentation` also uses both of those.

fwiw, i noticed the `Changes to openapi.yml require updating openapi.yml for sdr-api and dor-services-app and generating models - see README` admonishment in the PR template, ran `exe/generator generate`, and this line dropped the `.optional`, turning into `attribute? :languageTag, Types::Strict::String`.  and the tests (which check that the field is optional) still passed. 🤷 (running that was also how i noticed that the comment here comes from the `openapi.yml` param `description`, so i lined those up to keep the diff between `exe/generator generate` and this PR to a minimum).